rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/client/log/LogParams.java

```diff
@@ -45,6 +51,7 @@ public LogParams(LogLevel level, @Nullable String message, @Nullable String conf
     this.threadName = threadName;
     this.loggerName = loggerName;
     this.stackTrace = stackTrace;
+    this.loggedAt = Instant.now();
```

Comment:
If you want the timestamp, I suggest to not collect it here, but make it a parameter, since logback can already provide it. See in `SonarLintRpcClientLogbackAppender::append`, you can get the log timestamp from `eventObject.getTimeStamp()`

Reply:
Makes sense, will change that